helpers/logging_iterator_wrapper.js

```diff
@@ -22,7 +22,13 @@ module.exports = (prefix, valueByKey, iteratee) => {
     numberOfKeysDone += 1;
 
     if (Date.now() - lastLogAt > interval && process.env.TEST === undefined) {
```

Comment:
For `&&` its better to compute further because if the first "test" is false, nodeJs will not look at the rest of the boolean.
```suggestion
    if (!process.env.TEST && Date.now() - lastLogAt > interval) {
```